odysseus/compose.yml

```diff
@@ -17,7 +17,13 @@ services:
         environment:
             - BR2_DL_DIR=/home/odysseus/shared_data/dl 
             - BR2_CCACHE_DIR=/home/odysseus/shared_data/ccache
+        secrets:
+            - ody_passwords
```

Comment:
tbh this might be a good method of storing secrets via Docker. We could create a GitHub Action that uses the GitHub Secrets directory that builds and pushes a Docker container which is used as the base for the local compose

Reply:
Wouldn't the docker image then contain the passwords?

Reply:
We kinda discussed in slack, but if we had a secret "build key" that we could lock passwords behind that people just had to know that might work best